src/components/PlioQuestion.vue

```diff
@@ -29,14 +28,16 @@
                 >
                   <!-- adding <label> so that touch input is just
                   not limited to the radio button -->
-                  <label>
+                  <label class="label">
                     <input
                       type="radio"
                       name="options"
                       v-model="selectedOption"
                       :value="option"
+                      :id = "'option_input_' + index"
                       @click="selectOption(index)"
-                    />{{ option }}
+                    />
+                      <div class="option_render" :id= "option" v-html="optionText[index]"></div>
```

Comment:
As explained for the `question` part, similarly, for the `option`, the text will be rendered as a HTML

Reply:
OPTIONAL: Just a style suggestion. Can we rename `index` to `optionNumber`?

Reply:
OPTIONAL style suggestion: Maybe rename `index` to `optionNumber`?